extras/docker/scripts/w3af_api_docker

```diff
@@ -15,6 +15,9 @@ def parse_args():
                                             ' is stable, try unstable to get'
                                             ' the latest w3af features.',
                         required=False)
+    parser.add_argument('-c', '--config-file',
+                        help='[Required] Path to API config file in YAML format.',
```

Comment:
Remove the `[Required]`, argparse will show the required arguments in a different way to the user so this text is redundant